modules/auxiliary/gather/qnap_backtrace_admin_hash.rb

```diff
@@ -90,11 +90,10 @@ def run
 
     if admin_hash
       print_good("Hopefully this is your hash: #{admin_hash}")
-      report_note(
-        host: rhost,
-        port: rport,
-        type: 'qnap.admin.hash',
-        data: admin_hash
+      store_valid_credential(
```

Comment:
A login indicates the credential object can actually log in to the specified service. But service information can also be used in the origin which says where the credential came from. It looks like this new method that has been added does not account for a situation where you don't want a login to be created but it's a service origin. I would recommend actually using the create_credential method instead here